pandas_bokeh/geoplot.py

```diff
@@ -143,6 +147,17 @@ def convert_geoDataFrame_to_patches(gdf, geometry_column_name="geometry"):
     return gdf_new
 
 
+def get_tick_formatter(formatter_arg):
+
+    if isinstance(formatter_arg, TickFormatter):
```

Comment:
Shouldn’t this be issubclass here?

Reply:
no, because `issubclass` requires the argument to be a class, so if you pass in e.g. `issubclass("0.0a", TickFormatter)` it will throw a `TypeError issubclass() arg 1 must be a class`. However, `isinstance(formatter_arg, TickFormatter)` works
<img width="852" alt="Screenshot 2019-05-27 at 20 48 05" src="https://user-images.githubusercontent.com/6398157/58437191-c34a1880-80c0-11e9-838c-84ddd7ca12a8.png">

Reply:
Yes, but you can use:

`issubclass(a.__class__, TickFormatter)`

Just a matter of taste. I just think that with issubclass it is a bit clearer what happens here. Do whatever you like 😉

Reply:
👍 sure. Have changed it to issubclass.